api/provision/mock-invocation-fn.ts

```diff
@@ -61,7 +75,8 @@ export function createCspResponse(request: ProvisionRequest): CspResponse {
       };
       console.log("Failed response : " + JSON.stringify(response));
```

Comment:
I noticed you were replacing `console.log` with `logger.info` was there a reason some of the `console.log` couldn't also be changed?